java/code/src/com/suse/manager/webui/controllers/DownloadController.java

```diff
@@ -520,8 +520,6 @@ private static void validateToken(String token, String channel, String filename)
             Opt.consume(channelClaim, () -> {
                 log.info(String.format("Token %s does provide access to any channel", token));
             }, channels -> {
-                log.info(String.format(
-                        "Token %s provides access to channels %s", token, String.join(",", channels)));
                 if (!channels.contains(channel)) {
```

Comment:
The idea was to print the info only in case of the result of a "forbidden" response.
Just move it inside the "if" statement.